src/ScatterPlot/ScatterPlot.jsx

```diff
@@ -1,14 +1,28 @@
-import React, {Component} from 'react'
+import React, {Component, PropTypes} from 'react'
 import Chart, {DefaultChart} from '../Chart'
 import {Scatter} from 'safe-framework-react-chartjs'
 
 @Chart
 class ScatterPlot extends Component {
   static propTypes = {
-    ...Chart.propTypes
+    ...Chart.propTypes,
```

Comment:
I don't believe this is necessary. Since Chart gets created/rendered, it will do its own PropType validation